hub/gads-ui/src/components/Admin/Users/UsersAdministration.js

```diff
@@ -136,6 +184,41 @@ function NewUser({ handleGetUserData }) {
                             <MenuItem value='admin'>Admin</MenuItem>
                         </TextField>
                     </FormControl>
+                    <FormControl fullWidth required>
```

Comment:
I wonder if something like [this](https://mui.com/material-ui/react-select/#checkmarks) wouldn't be more appropriate? What do you think?

Reply:
Yeah, you are right! The user experience will be better, and the UI will fit better with the Select element as more workspaces are selected. Done in this [commit](https://github.com/gifflet/GADS/commit/11b90bb1b1a48c27197ab595fb27b23ac27149ce).